gulpfile.js

```diff
@@ -12,5 +12,12 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
+
+    // run: gulp && gulp watch | php artisan serve --host=0.0.0.0
+    mix.browserSync({
```

Comment:
For now, let's un-do this and make it a separate pull request so we can review it separately. Thanks!